src/olympia/promoted/migrations/0029_auto_20250429_1315.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.20 on 2025-04-29 13:15
+
+from django.db import migrations
+
+from olympia.constants.promoted import PROMOTED_GROUP_CHOICES
+
+def set_partner_group_high_profile_rating(apps, schema_editor):
+    PromotedGroup = apps.get_model('promoted', 'PromotedGroup')
+    group = PromotedGroup.objects.get(group_id=PROMOTED_GROUP_CHOICES.PARTNER)
```

Comment:
is it possible for the situation to occur where this group does not exist? If so, you'll need to handle it by wrapping in a try (or otherwise handle it in a safe way).  Similarly with the reverse - database migrations should never fail.

Reply:
Hm, it shouldn't with the dependencies, but doesn't hurt.